packages/wdio-jasmine-framework/src/reporter.js

```diff
@@ -19,6 +19,11 @@ export default class JasmineReporter {
         suite.start = new Date()
 
         this.emit('suite:start', suite)
+
+        this.emitHookStart(suite, 'before all')
```

Comment:
I still think that it, as well as emit hook end, can be moved to `if (suite.failedExpectations.length) {` block, so that we only add failed hooks to report